### ISSUE TITLE
Alt+Space doesn't work from Settings pages

### ISSUE THREAD

**Issue by DHowett** [microsoft/terminal#14202]
Accessibility issue. Cannot access system menu from keyboard.

Open Terminal to say a command prompt. Hit Alt+Space to open the system menu, and then pick "Settings". Now hit Alt+Space to open the system menu again, say because you want to maximize the terminal.

Expect: System menu opens.
Actual: Nothing happens.

The only way to get Alt+Space to work is to click on a blank space on the title bar (to get focus out of the Settings pane). However, there is no way to get focus onto the title bar via the keyboard. Alt+Up can't go up that high.

Alt+Space is a very common keyboard shortcut and users expect it to work universally. Requiring a click on the title bar defeats the purpose of a keyboard shortcut.

**Comment by DHowett** [comment]
Originally filed in MSFT-41390832

**Comment by zadjii** [comment]
#11970
